content/en/reference/deployment/config-file/targets.md

```diff
@@ -117,9 +120,28 @@ targets:
 
 #### `targets.<targetName>.constraints.beforeDeployment`
 
-Conditions that must be met before the deployment can start. These are in addition to the deployments you define in `dependsOn` that must finish.
+Add conditions that must be met before the deployment can start. These are in addition to the deployments you define in `dependsOn` that must finish.
 
-You can specify a pause that waits for a manual approval or a certain amount of time before starting.
+You can specify a pause that waits for a manual approval or a certain amount of time before starting. You can additionally add a webhook to the `beforeDeployment` constraint.
```

Comment:
a user can add pause (timed or manual / with or without role), webhook and analysis as part of beforeDeployment and afterDeployment constraint.